### PR TITLE
Always Generate r6/cache/modded/ When Installing REDmods

### PR DESCRIPTION
**Pull request by effs**
Why not?

### CODE REVIEW

File: src/installers.ts [hunk 1]
```diff
@@ -458,14 +462,20 @@ const giftwrapSourcesAgainIfNecessary = (
...
+      instructions,
+      map((instruction) => (
+        instruction.source
+          ? {
+            ...instruction,
+            source: path.join(treeInUse.wrapperDir, instruction.source),
+          }
+          : instruction
+      )),
+    ));
```
Comment (E1337Kat) [inline src/installers.ts:478]: Could you walk me through this change? I don't quite see how it works, type wise I mean.

Comment (E1337Kat) [reply src/installers.ts:478]: does 
```js
{
  ...instruction,
  source: path.join(treeInUse.wrapperDir, instruction.source),
}
```
 basically unwrap `instruction` and then replace the `source` field of the object?

**Review by E1337Kat — APPROVED** [review]
I am curious about that object update, but otherwise this looks good.

File: src/installers.ts (hunk 1 above)
Comment (effs) [reply src/installers.ts:478]: Yes, it's like a chained `Object.assign`

Comment (effs) [reply src/installers.ts:478]: (And without having to worry about which other fields might exist.)

**Pull request merged**